maud/src/lib.rs

```diff
@@ -168,11 +179,11 @@ pub const DOCTYPE: PreEscaped<&'static str> = PreEscaped("<!DOCTYPE html>");
 #[cfg(feature = "iron")]
 mod iron_support {
     use crate::PreEscaped;
+    use core::io;
```

Comment:
Does this work?

I'm pretty sure `core` doesn't have an `io` module.

You might want to disable `no_std` when these web framework features are enabled, since they require `std` anyway.

Reply:
I didn't find a way to optionally disable `no_std`, only a way to optionally enable it.